force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -230,33 +278,49 @@ public with sharing class ClassesObjectsMethods {
         // title
         // author
         // pages
-    
+        public String title;
+        public String author;
+        public Integer pages;
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(String title, String author) {
             // Initialize the title and author
+            this.title = title;
+            this.author = author;
+        }
+        public Book(String title, String author, Integer pages) {
+            this.title = title;
+            this.author = author;
+            this.pages = pages;
+        }
+
+        public Book(String title) {
+            this.title = title;
```

Comment:
These 3 different instances of Book might be confusing when you call the book class. Try having one instance for this exercise